lib/app/console_app.rb

```diff
@@ -129,13 +129,13 @@ def connector_registered?(warn_if_not: true)
     end
 
     def create_connector(index_name, force: false)
-      connector_settings = Core::ConnectorSettings.fetch(connector_id)
-
-      if connector_settings.nil? || force
-        created_id = Core::ElasticConnectorActions.create_connector(index_name, App::Config['service_type'])
-        connector_settings = Core::ConnectorSettings.fetch(created_id)
+      id = connector_id
+      if Core::ElasticConnectorActions.get_connector(connector_id)[:found] == false || force
+        id = Core::ElasticConnectorActions.create_connector(index_name, App::Config['service_type'])
```

Comment:
If the connector exists and `force = true`, it will try to create the connector in `.elastic-connectors`. It won't raise errors but we should have the restriction at the application level to avoid duplicate index names.

It will also bring problems that two connectors write data to the same content index.

Reply:
That is true, however this PR just adds a human-readable error message, not changing the current flow.

This comment should be addressed in a different PR cc @mchernyavskaya